### DISCUSSION TITLE
Google Test dependency

### DISCUSSION THREAD

**Discussion by ramcdona** [elalish/manifold#552] [General]
It seems that building with tests on depends on the GoogleTest framework.  From what I can tell, this isn't documented anywhere or in the repository.  I suspect it is just taken for granted by the core devs.

**Comment by elalish** [comment]
True, hopefully not a problem? Our CMake now takes care of it automatically if you don't have it installed. Would you like to update our README?

**Comment by ramcdona** [comment]
No problem.

I don't have it installed, but I think CMake found a version installed with Anaconda -- which I think was an X86 version.  I'm building on an M1 Mac, so it got through the compile (the header worked fine), but when it got to the link step, it couldn't find the desired symbols.

I'll try to find your CMake solution to turn them off if it isn't installed -- and see if I can tweak it to prevent this kind of mess.

I'm fine running without tests, but figured I would give them a try to start with.
